README.md

```diff
@@ -141,7 +141,7 @@ For convenience, you may desire to house the `npx` command within an npm script
 
 ```json
   "scripts": {
-    "last-failed": "npx cypress-run-last-failed run --e2e --browser electron"
+    "last-failed": "cd ${failedTestDirectory} && npx cypress-plugin-last-failed run --e2e --browser electron"
```

Comment:
Could you help me understand how the script here will read and understand `${failedTestDirectory}`?

Reply:
It does not pretend to the script to understand it as a variable; it should be settled up by the user with the correct directory, which is the same as you used as `failedTestDirectory`. How do you think we could indicate it better?